src/DynamoCoreWpf/VisualizationManager/VisualizationManager.cs

```diff
@@ -226,7 +226,7 @@ public void Start(bool update = false)
         /// <param name="path"></param>
         public void TagRenderPackageForPath(string path)
         {
-            var packages = new List<RenderPackage>();
+            var packages = new List<IRenderPackage>();
 
             //This also isn't thread safe
             foreach (var node in dynamoModel.CurrentWorkspace.Nodes)
```

Comment:
Looks like this will not interact well with multiple home workspaces.  Oh well...